packages/runtime-core/src/h.ts

```diff
@@ -159,3 +160,23 @@ export function h(type: any, propsOrChildren?: any, children?: any): VNode {
     return createVNode(type, propsOrChildren, children)
   }
 }
+
+const originalH: Function = exports.h
```

Comment:
`exports` is only available in Node.js environments (the tests pass because `jest` runs the tests in Node.js) and won't work when bundled.

What we should here is let `transformHArgs` save the transformer in a local variable, and check for the existence of that variable inside `h`. We should probably also wrap the check in a `if (__DEV__)` block.

Reply:
Right, that is what I was thinking as well. So something like:

```
let tranformedArgs

export function tranformHArgs(...args) {
  tranformedArgs = [...args]
}

export function resetHArgs(...args) {
  tranformedArgs = undefined
}

// other stuff

export function h(type: any, propsOrChildren?: any, children?: any): VNode {
  if (__DEV___ || __TEST__ && transformedArgs) {
    return createVNode(...transformedArgs)
  }

  // current implementation 
  // ...
}
```

Is what you are suggesting @yyx990803 ?

This will have a limitation we cannot run many tests in parallel, since they all share the same state. I think we might just have to live with that, I don't see another easy option that doesn't involve a lot of changes. 

Thanks for the feedback.

Reply:
Well I was thinking of something more like
``` js
let argsTransformer

export function transformHArgs(transformer) {
  argsTransformer = transformer
}

function originalH(...) {}

export const h = __DEV__
  ? (...args) => {
    if (argsTransformer) {
      args = argsTransformer(args, currentRenderingInstance)
    }
    return originalH(...args)
  }
  : originalH
```

Reply:
Aha! V-nice, will update ⚡️